### DISCUSSION TITLE
rickroll in the website

### DISCUSSION THREAD

**Discussion by FunctionalMetatable** [CluckCluckChicken/Magnifier#35] [big chungus]
lol

**Comment by JackyBorderCollie** [comment]
yes

**Reply by davwat** [reply]
view the very legitimate preview at [https://magnifier.web.app/earlyaccessgivemefreevbuxplsilovefortniteisagenuinelyentertaininggamethatisntjustaboutshootingsmallchildren](https://magnifier.web.app/earlyaccessgivemefreevbuxplsilovefortniteisagenuinelyentertaininggamethatisntjustaboutshootingsmallchildren)

**Reply by aidan-crockett** [reply]
oooh very nice

**Reply by CST1229** [reply]
[this still exists on release magnifier!](https://magnifier.potatophant.net/earlyaccessgivemefreevbuxplsilovefortniteisagenuinelyentertaininggamethatisntjustaboutshootingsmallchildren)

**Reply by JackyBorderCollie** [reply]
noice
